app/models/role.rb

```diff
@@ -75,6 +75,27 @@ class Role < ApplicationRecord
   scoped_search :on => :builtin, :complete_value => { :true => true, :false => false }
   scoped_search :on => :description, :complete_value => false
 
+  class << self
+    attr_accessor :bypass_locking
```

Comment:
could it perhaps be wrapped in a class method accepting block like the instance method does? that could contain ensure block with resetting the bypass, ideally not setting it to false but to $original_value in case someone has two bypasses nested

and I'd prefer if the name would be consistent with instance variant which is called `ignore_locking`